package.json

```diff
@@ -21,12 +21,17 @@
         "ecosystem:phonegap",
         "cordova-android"
     ],
-    "engines": [
-        {
-            "name": "cordova",
-            "version": ">=5.0.0"
+    "engines": {
+      "cordovaDependencies": {
+        "<1.2.0": {
+          "cordova": ">=5.0.0"
+        },
+        "1.2.0": {
```

Comment:
Yes, I suppose it can be. I just assume that 1.2.0 would be the last version of this plugin ever.